en/docs/administer/key-managers/configure-forgerock-connector.md

```diff
@@ -0,0 +1,288 @@
+# Configure Forgerock as a Key Manager
```

Comment:
"Forgerock" should be written as "ForgeRock" based on the way it's own company writes it.